content/docs/user-guide/setup-google-drive-remote.md

```diff
@@ -204,8 +205,13 @@ commit** this file with Git. Instead, add it into `.gitignore` and never share
 it with other people.
 
 If you wish to change the user you have authenticated with, or for
-troubleshooting misc. token errors, simply remove the user credentials JSON file
-and authorize again.
+troubleshooting misc. token errors, you can remove the user credentials file
+and authorize again. Alternatively you can use a custom `profile`-based
+credential cache (instead of the default, `gdrive_client_id`-based one):
```

Comment:
Format
```suggestion
troubleshooting misc. token errors, you can remove the user credentials file and
authorize again. Alternatively you can use a custom `profile`-based credential
cache (instead of the default, `gdrive_client_id`-based one):
```